cnho_stables/chain.json

```diff
@@ -0,0 +1,112 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "cnho stables",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://cnho.io/",
+  "pretty_name": "cnho stables",
+  "chain_id": "cnho_stables-1",
+  "bech32_prefix": "cnho",
```

Comment:
Please register this prefix: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
i remove the field of bech32_prefix and is ok?

Reply:
It is registered.